### ISSUE TITLE
A mistake about  dim_Model.py ?

### ISSUE THREAD

**Issue by HellFireC02** [Cory-M/DCCM#6]
Sorry to bother you, but i found a contradiction in your code.
I read your paper, and find the difference between the E.q(14) and your actual code in dim_Model.py.

In your dim_Model.py, your code is:
**# local loss
Ej = -F.softplus(-self.local_D(Y_cat_M)).mean()
Em = -F.softplus(self.local_D(Y_cat_M_fake)).mean()
local_loss = -(Em + Ej)**

but for E.q(14), it should be:
**local_loss = -( Ej-Em)**

Is some thing wrong in my analysis?
Or it's an error in your code?
Looking forward to your reply!

**Comment by Cory-M** [comment]
Hi, thanks for pointing that out! I checked the code and realized I made a notation mistake for Em. The Em I wrote in the code before was actually -Em. I've corrected it already, please feel free to check the changelog for that. 
Let me know if further clarification need to be made - thanks again!

**Comment by HellFireC02** [comment]
Thanks for your introduction.
If I encounter other problems with code, I will contact you in time : D
